using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md

```diff
@@ -0,0 +1,417 @@
+
+![Adobe Journey Optimizer - Decisioning extension configuration](../../.gitbook/assets/ajo-decisioning-extension-configuration.png)
+
+## Integrate Experience Platform Optimize SDK in your mobile application
+
+{% hint style="warning" %}
+For AEPOptimize APIs to work properly, it is required that you integrate Mobile Core and Edge extensions as well in your mobile app. For more details see, [Mobile Core](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core) and [Adobe Experience Platform Edge Network](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension).
+{% endhint %}
+
+### Install the Experience Platform Mobile SDK
+
+{% tabs %}
+{% tab title="iOS - Swift" %}
+1. Configure your app target to fetch Mobile Core, Edge, Identity for Edge Network, Optimize from Cocoapods by specifying the following pod dependencies in your `Podfile`.
+
+   ```swift
+   platform :ios, '10.0'
+
+   use_frameworks!
+   target 'YourAppTarget' do
+       pod 'AEPCore'
+       pod 'AEPEdge'
+       pod 'AEPEdgeIdentity'
+       pod 'AEPOptimize'
+   end
+   ```
+{% endtab %}
+{% endtabs %}
+
+### Register the extensions with Mobile Core
+
+{% tabs %}
+{% tab title="iOS - Swift" %}
+#### Swift
+
+```swift
+// AppDelegate.swift
+
+import AEPCore
+import AEPEdge
+import AEPEdgeIdentity
+import AEPOptimize
+
+@UIApplicationMain
+final class AppDelegate: UIResponder, UIApplicationDelegate {
+  var window: UIWindow?
+
+  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool {
+
+      // register the extensions
+      MobileCore.registerExtensions([Edge.self, AEPEdgeIdentity.Identity.self, Optimize.self]) {
+        MobileCore.configureWith(appId: YOUR_ENVIRONMENT_FILE_ID)
+      }
+
+      return true
+  }
+}
+```
+
+#### Objective-C
+
+```objc
+// AppDelegate.m
+
+@import AEPCore;
+@import AEPEdge;
+@import AEPEdgeIdentity;
+@import AEPOptimize;
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+
+    // register the extensions
+    [AEPMobileCore registerExtensions:@[AEPMobileEdge.class, AEPMobileEdgeIdentity.class, , AEPMobileOptimize.class] completion:^{
+      [AEPMobileCore configureWithAppId: YOUR_ENVIRONMENT_FILE_ID];
+  }];
+  ...
+}
+```
+{% endtab %}
+{% endtabs %}
+
+## Adobe Journey Optimizer - Offer Decisions
+
+{% hint style="warning" %}
+Some offer constraints are currently not supported with the mobile Experience Edge workflows, for example Capping. The Capping field value specifies the number of times an offer can be presented across all users. For more details, see [Offer eligibility rules and constraints guide](https://experienceleague.adobe.com/docs/offer-decisioning/using/managing-offers-in-the-offer-library/creating-personalized-offers.html#eligibility).
+{% endhint %}
+
+### Decision Scope
```

Comment:
I'd prefer `Decision Scope`, since I'm trying to explain the concept here. Please refer to this guide - https://experienceleague.adobe.com/docs/experience-platform/edge/personalization/offer-decisioning/offer-decisioning-overview.html?lang=en
I have a separate class reference for the `DecisionScope` class in APi reference